app/models/spree_avatax_certified/address.rb

```diff
@@ -45,8 +45,11 @@ def ship_to_address
     end
 
     def ship_from_address
-      package = Spree::Stock::Coordinator.new(@order).packages.find { |package| package.line_items.any? {|li| li.id == @line.id} }
-      stock_location = package.stock_location
+      stock_location = @line.stock_location
+      unless stock_location
```

Comment:
Nitpick: could be extracted to a method like:
`get_stock_location unless stock_location.present?`